client/src/html.tsx

```diff
@@ -1,4 +1,18 @@
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
 import React from 'react';
+import { sentry } from '../../config/secrets';
+
+if (sentry.dns === 'dsn_from_sentry_dashboard') {
+  console.log('Sentry reporting disabled unless DSN is provided!');
+} else {
+  Sentry.init({
+    dsn: sentry.dns,
+    integrations: [new Integrations.BrowserTracing()],
+    tracesSampleRate: 1.0
+  });
+  console.log('Sentry initialized');
```

Comment:
Do we want to be using the same DSN for both the client and API?

Reply:
Good catch. No, that would be very confusing.

I've set up a new project and I'll make that key available to the client.